l3kernel/l3pdf.dtx

```diff
@@ -101,8 +101,8 @@
 %
 % \subsection{Indexed objects}
 %
-% Objects can be created using a \meta{class} and \emph{index}; the
-% \meta{class} argument should expand to character tokens, whilst the
+% Objects can also be created using a pair of \meta{class} and \emph{index};
```

Comment:
This does not look quite right: 
Is the “also” needed here (not looked at the previous sentence sentences)? 
Bette:
. . . using a class, index pair, as follows: . . .

Reply:
> Is the “also” needed here (not looked at the previous sentence sentences)?

Yes, as this is the second way (`\subsection{Indexes objects}`) to create PDF objects. The first way is named objects (`\subsection{Named objects}`), see line 54 of `l3kernel/l3pdf.dtx` in this PR.

Reply:
Yes, I see the reasoning now.
But in fact “indexed” are just a special case of “named” so the whole se timon needs rejigging.

Reply:
'Named' are entirely arbitrary: better ways of phrasing welcome